nengo_loihi/hardware/interface.py

```diff
@@ -153,6 +166,23 @@ def run_steps(self, steps, blocking=True):
         # start the board running the desired number of steps
         d_get(self.nxsdk_board, b"cnVu")(steps, **{d(b"YVN5bmM="): not blocking})
 
+        # connect to host socket
+        if self.host_socket is not None and not self.host_socket_connected:
+            # pause to allow host snip to start and listen for connection
+            time.sleep(0.1)
+
+            host_address = self.nxsdk_board.executor._host_coordinator.hostAddr
```

Comment:
I think we discussed this to check if the `nxsdk` attributes need to be obfuscated or not

Reply:
We should probably obfuscate just to be safe and consistent (though the main point of obfuscation is to stop stuff coming up in Google, and I don't think anybody is going to be Googling this).